Change default seeding strategy to have fast seeds (#14985)
decidim/decidim: docs/modules/configure/pages/environment_variables.adoc

|5
|No

|SLOW_SEEDS

> |If seeding various environment that needs to have more data. Please use `SLOW_SEEDS` environment variable to add additional seed data.

Author: ```suggestion
|If seeding an environment that needs to have more data, please use `SLOW_SEEDS` environment variable to add it.
```
